packages/editor-ui/src/views/SettingsPersonalView.vue

```diff
@@ -215,7 +216,7 @@ async function onMfaEnableClick() {
 			type: 'error',
 		});
 		try {
-			await usersStore.confirmEmail();
+			await usersStore.sendConfirmationEmail();
 		} catch {}
```

Comment:
Let's not swallow error like this. This way it's not reported to sentry either and can hide bugs.